lib/customErrorHandler.js

```diff
@@ -3,14 +3,19 @@ const debug = require('debug')('swagger-endpoint-validator:custom-error-handler'
 /**
  * Adds a custom error handler for the errors thrown by the validator.
  * @param {Object} app - Express application object.
+ * @param {Object} options - Configuration options.
  * @see https://github.com/cdimascio/express-openapi-validator#asyncawait
  */
-const add = app => {
+const add = (app, options) => {
```

Comment:
```suggestion
const add = (app, { propagatedError }) => {
```
I'm not sure if this can make the code more readable.

Reply:
Sure. However, if we keep adding more options, it could be a bit cumbersome